src/ioPlacer/src/DBWrapper.cpp

```diff
@@ -249,29 +245,25 @@ void DBWrapper::commitIOPlacementToDB(std::vector<IOPin>& assignment)
 {
   odb::dbBlock* block = _chip->getBlock();
   if (!block) {
-    std::cout << "[ERROR] odb::dbBlock not found! Exiting...\n";
-    std::exit(1);
+    error("odb::dbBlock not found\n");
   }
 
   odb::dbTech* tech = _db->getTech();
   if (!tech) {
-    std::cout << "[ERROR] odb::dbTech not initialized! Exiting...\n";
-    std::exit(1);
+    error("odb::dbTech not initialized\n");
   }
 
   int horLayerIdx = _parms->getHorizontalMetalLayer();
   int verLayerIdx = _parms->getVerticalMetalLayer();
 
   odb::dbTechLayer* horLayer = tech->findRoutingLayer(horLayerIdx);
   if (!horLayer) {
-    std::cout << "[ERROR] Layer" << horLayerIdx << " not found! Exiting...\n";
-    std::exit(1);
+    error("Layer %d not found\n", horLayerIdx);
```

Comment:
I think these checks would make more sense in tcl. the idea is to do error checking at the command level so you don't need throws in the c++